### ISSUE TITLE
a question

### ISSUE THREAD

**Issue by shenyujie1125** [ilucasgoncalves/AuxFormer#4]
hello, when you trained your AUXFORMER model with two modalities and test your model with a single modality(another modality is zero-padded),  which branch results did you choose when testing?  
the result from the main branch?  
or the auxiliary branch corresponding to the input modality in the test period?
thankyou!

**Comment by shenyujie1125** [comment]
for example, when you input video to your model with audio zero-padded, did you choose the result of the auxiliary video branch to compute accuracy?
or when you input video to your model with audio zero-padded, did you choose the result of the fusion branch to compute accuracy?

**Comment by ilucasgoncalves** [comment]
All branches contribute to the prediction at all times (full audiovisual, audio only with video set to zeroes, and video only with audios set to zeroes). 

Check the file ```test.py``` lines 112 and 113. 

```
112      preds_va, preds_a, preds_v = modelWrapper.feed_forward(xa, xv, attention_mask=mask)
113      pred = (preds_va + preds_a + preds_v) / 3
```
